longevity_test.py

```diff
@@ -33,12 +33,11 @@ class LongevityTest(ClusterTester, loader_utils.LoaderUtilsMixin):
 
     default_params = {'timeout': 650000}
 
-    def _get_scan_operation_params(self, scan_operation: str) -> dict:
-        params = {}
-        if scan_operation_params := self.params.get(scan_operation):
-            params = json.loads(scan_operation_params)
-            self.log.info('Scan operation %s params are: %s', scan_operation, params)
-        return params
+    def _get_scan_operation_params(self) -> list[dict]:
+        params = self.params.get("run_fullscan") if self.params.get("run_fullscan") else {}
```

Comment:
Unfortunately not - this is a very deceptive change in `SCTConfiguration` which overloaded the `get` method without implementing a default return optional argument, so the proposed solution would fail. I can adjust `SCTConfugration`'s implementation of the getter if don't forget.

Reply:
i'm not sure i understand, because `get` doesn't need to return you any default... you are setting the default, in the 2nd argument in the `get` call... but perhaps i don't fully understand the issue you say (and IIRC we don't want to have defaults in `SCTConfiguration` intentionally)

Reply:
You mean the `get` for a regular `dict`, this `get` is given the override treatment here: https://github.com/scylladb/scylla-cluster-tests/blob/98099ceb8e2cf4b10c554fa0b1274d7547b5ff33/sdcm/sct_config.py#L1773-L1786 So it takes only one argument.